core/src/main/scala/kafka/server/ReplicaFetcherThread.scala

```diff
@@ -63,7 +65,8 @@ class ReplicaFetcherThread(name: String,
 
   // Visible for testing
   private[server] val fetchRequestVersion: Short =
-    if (brokerConfig.interBrokerProtocolVersion >= KAFKA_2_0_IV1) 8
+    if (brokerConfig.interBrokerProtocolVersion >= KAFKA_2_0_IV1) 9
```

Comment:
Should this be `KAFKA_2_1_IV1`? Same for `offsetForLeaderEpochRequestVersion` and `listOffsetRequestVersion`.

Reply:
Hmm... it seems that `KAFKA_2_0_IV1` is still used for `offsetForLeaderEpochRequestVersion` and `listOffsetRequestVersion`. Could you fix them as well?

Reply:
I made a commit to fix it :)